libnethack/src/sounds.c

```diff
@@ -810,6 +810,26 @@ domonnoise(struct monst *mtmp)
         else
             verbl_msg = "Who do you think you are, War?";
         break;
+    case MS_RTP:
+        {
+            static const char *const rtp_foe_msg[3] = {
+                "What about our imagine subscription?",
+                "I can ensure everything is down...",
+                "See if you can go to sleep now.",
+            }, *const rtp_pax_msg[8] = {
+                "Not my server, not my problem.",
+                "You name it, we don't have to vote on it.",
+                "I can get it ALL on it!",
+                "Yeah but that definitely happened again.",
+                "Yeah I'd like to think about what we want",
+                "Why did MegaVM start VMs wtf!? Welp... shit shit shit WHO was last logged in?",
+                "I wasn't saturating the upload on 49net...",
+                "I can't remember how I fixed it.",
+            };
+            verbl_msg = mtmp->mpeaceful ? rtp_pax_msg[rn2(8)]
```

Comment:
Use `rn2(sizeof(rtp_pax_msg) - 1)` to infer the size of the array automatically. Also create separate variables for the size of the two arrays so it's clear to the reader what's going on.

Reply:
I fixed this behavior to use `sizeof` as you recommend, but it's important to note that the input value to `rn2` is `max_plus_1` so `rn2(10)` can only return results within the range 0-9.